bbc/simorgh
# Add 3rd party links to Hindi podcasts, update Persian and Somali navigation

Resolves JIRA n/a

Overall changes
======
Addeed third party links to the newest Hindi podcasts, Update Persian podcast to remove link to 2022/23 protests and move radio link to last position of navigation. Updated Somali nav to remove last item as that content is no longer updated.

Code changes
======

- Added third party links for podcastsd p0fmrg25 and p0frdsxv to /src/app/routes/onDemandAudio/tempData/podcastExternalLinks/hindi.js
- Edited Navigation section in /src/app/lib/config/services/persian.ts to remove second item and move radio link (رادیو)
to the last position
- edited Navigation section in /src/app/lib/config/services/persian.ts to remove last item (Barnaamijka Baafinta)
- updated snapshots

Testing
======
1.  Open /hindi/podcasts/p0fmrg25 third party links section shoudl include links to Spotify,Apple and Ganaa
2.  Open /hindi/podcasts/p0frdsxv third party links section shoudl include links to Spotify and Apple
3. Open BBC Persian front page (/persian) the second item in the navigation should point to /persian/media-49522521,
 the last link shoudl point to /persian/tv-and-radio-37434376
4 Open the Somali front page, the navigation should contain only five items (War | Ganacsi | Cayaaraha | Muuqaal | Barnaamijyada Idaacadda)

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
